okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/OkHttpClients.java

```diff
@@ -130,8 +130,16 @@ private static RemotingOkHttpClient createInternal(
             // TODO(rfink): Should this go into the call itself?
             client.addInterceptor(new MeshProxyInterceptor(config.meshProxy().get()));
         } else {
-            // Add CurrentUrlInterceptor: always selects the "current" URL, rather than the one specified in the request
-            client.addInterceptor(CurrentUrlInterceptor.create(urlSelector));
+            switch (config.nodeSelectionStrategy()) {
+                case ROUND_ROBIN:
+                    client.addInterceptor(RoundRobinUrlInterceptor.create(urlSelector));
+                    break;
+                case PIN_UNTIL_ERROR:
+                default:
```

Comment:
let's remove the `default`. error-prone will flag if this switch is incomplete. (can you verify, please?)